affiliates/colombia.swagger.json

```diff
@@ -0,0 +1,410 @@
+{
+  "openapi" : "3.0.0",
+  "info" : {
+    "version" : "3.0.0",
+    "title" : "Colombia docs",
```

Comment:
should be "Colombia affiliates API"

Reply:
other countries have "Countryname docs" should I change it there as well?

Reply:
https://documentation.credy.com/docs/mexico-affiliates

But no, not right now